tests/cursorid/cursorid-set_state-001.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+MongoDB\Driver\CursorId::__set_state()
+--FILE--
+<?php
+
+$cursorId = unserialize('C:23:"MongoDB\Driver\CursorId":42:{a:1:{s:2:"id";s:19:"7250031947823432848";}}');
+
+var_export(MongoDB\Driver\CursorId::__set_state([
+    'id' => "$cursorId",
+]));
+echo "\n\n";
```

Comment:
A single newline is fine. The original tests used two just to add more space between each successive `var_export` call (assuming this was lifted from the ReadConcern tests, with multiple cases in a single phpt file).

If you remove the newline, you can adjust the expected output accordingly.